languages/javascript/exercises/concept/closures/.meta/design.md

```diff
@@ -0,0 +1,89 @@
+This issue describes how to implement the `closures` concept exercise for the `javascript` track.
+
+## Goal
+
+The goal of this exercise is to teach the student how _closures_ are implemented in `JavaScript`.
+
+In other words: how _values_ can be _enclosed_ in a _function_. It touches onto `scoping` and `functions` in general, and is usually used as `higher-order-functions`, `callbacks` or to change `visibility` (think: private).
+
+## JavaScript specificness
```

Comment:
I'm wondering if this should be in the design document, or maybe in a reference document (see https://github.com/exercism/v3/tree/master/languages/javascript/reference). I'd prefer the `design.md` document to be as concise as possible.

Reply:
It was something I copied from the issue.  I can remove it.